docs/styleguide/PyGuide.md

```diff
@@ -965,6 +965,11 @@ Always use a `.py` filename extension. Never use dashes.
 Python filenames must have a `.py` extension and must not contain dashes (`-`).
 This allows them to be imported and unit tested.
 
+Avoid file and module names of `utils`, `helpers` etc. and do not extend existing files and packages with these names.
+Otherwise there is always a temptation to add an additional piece of code to this file/module, which ends up in having large files with unrelated functions and classes.
+Instead group your new code in dedicated files/modules that are named explicitly according to the purpose of code - either create new files/modules for this or extend the already existing, properly named ones.
+If you see an opportunity to break down a `utils.py` or `helpers.py` file into a better named and grouped set of files - take it.
```

Comment:
```suggestion
Avoid creating Python files like a "swiss knife" naming their with generic names such as `utils.py`, `helpers.py`, `common.py` and etc.
```

Reply:
Rewrote the paragraph, please re-review.

Reply:
Let's handle `*.py` at the first.